infrastructure/index.ts

```diff
@@ -106,26 +130,47 @@ const distributionArgs: aws.cloudfront.DistributionArgs = {
     defaultRootObject: "index.html",
 
     defaultCacheBehavior: {
-        targetOriginId: contentBucket.arn,
-        compress: true,
-
-        viewerProtocolPolicy: "redirect-to-https",
-
-        allowedMethods: ["GET", "HEAD", "OPTIONS"],
-        cachedMethods: ["GET", "HEAD", "OPTIONS"],
+        ...baseCacheBehavior
+    },
 
-        // S3 doesn't need take any of these values into account when serving content.
-        forwardedValues: {
-            cookies: {
-                forward: "none",
-            },
-            queryString: false,
+    orderedCacheBehaviors: [
+        {
```

Comment:
These `pathPattern` blocks are repeated because Amazon doesn't seem to support expressing multiple paths, e.g., as a real regular expression; they just provide for simple wildcards like `?` and `*`.